armi/nuclearDataIO/xsLibraries.py

```diff
@@ -200,8 +200,22 @@ def mergeXSLibrariesInWorkingDirectory(lib, xsLibrarySuffix="", mergeGammaLibs=F
                 for nuc in neutronLibrary.nuclides
                 if isinstance(nuc._base, nuclideBases.DummyNuclideBase)
             ]
+
+            gamisoLibraryPath = nuclearDataIO.getExpectedGAMISOFileName(suffix=xsLibrarySuffix, xsID=xsID)
+            pmatrxLibraryPath = nuclearDataIO.getExpectedPMATRXFileName(suffix=xsLibrarySuffix, xsID=xsID)
+
+            # Check if the gamiso and pmatrx data paths exist with the xs library suffix so that
+            # these are merged in. If they don't both exist then that is OK and we can just
+            # revert back to expecting the files just based on the XS ID.
+            if not (os.path.exists(gamisoLibraryPath) and os.path.exists(pmatrxLibraryPath)):
+                runLog.warning(f"Neither the GAMISO or PMATRX data exist for "
```

Comment:
"One of GAMISO and/or PMATRIX data does not exist for..."

"Neither" would apply to logic like `if not os.path.exists(gamiso) and not os.path.exists(pmatrx)`

Reply:
Updating now.